package.json

```diff
@@ -179,6 +179,11 @@
           "light": "assets/document-search-light.svg",
           "dark": "assets/document-search-dark.svg"
         }
+      },
+      {
+        "command": "dune.current-dune-file",
+        "category": "Dune",
+        "title": "Open Dune File (located in the same folder)"
```

Comment:
I think we should keep the command IDs under a consistent organization.

This should be `ocaml.current-dune-file` under the OCaml category so it's as discoverable as the other commands. The "Dune" keyword is already present in the command title, so the other parts are redundant.

Reply:
Oops, oversight on my part, I agree with @mnxn